graph/src/log/elastic.rs

```diff
@@ -266,28 +261,28 @@ impl ElasticDrain {
                     // Send batch of logs to Elasticsearch
                     let client = Client::new();
                     let logger_for_err = flush_logger.clone();
-                    Box::new(
-                        client
-                            .post(batch_url)
-                            .header("Content-Type", "application/json")
-                            .basic_auth(
-                                config.general.username.clone().unwrap_or("".into()),
-                                config.general.password.clone(),
-                            )
-                            .body(batch_body)
-                            .send()
-                            .and_then(|response| response.error_for_status())
-                            .map(|_| ())
-                            .map_err(move |e| {
-                                // Log if there was a problem sending the logs
-                                error!(
-                                    logger_for_err,
-                                    "Failed to send logs to Elasticsearch: {}", e
-                                );
-                            }),
-                    )
-                }),
-        );
+
+                    let _ = client
```

Comment:
I think it would be better to have a named variable for occurrences of `let _ =`. The other bug for unpolled futures in this PR makes me now nervous when seeing this construction. Better may be `let _ignore_result_and_continue =` or a comment or something which indicates the intent. Note also that `let _ignore_result` and `let _` have surprising semantic differences: https://github.com/rust-lang/rust/issues/10488

Reply:
I've refactored this so we don't need `let`. Not that it makes a difference in this case.